addrmgr/addrmanager.go

```diff
@@ -1130,6 +1155,39 @@ func (a *AddrManager) GetBestLocalAddress(remoteAddr *wire.NetAddress) *wire.Net
 	return bestAddress
 }
 
+// IsPeerNaValid asserts if the the provided network address is routable
+// and reachable from the peer that suggested it.
+func (a *AddrManager) IsPeerNaValid(na *wire.NetAddress, peer *wire.NetAddress) bool {
```

Comment:
Please rename the parameters as they are quite confusing as named.

I would suggest:
```
func (a *AddrManager) IsPeerNaValid(localAddr, remoteAddr *wire.NetAddress)
```